networkconfig/holesky-stage.go

```diff
@@ -16,8 +16,8 @@ var HoleskyStage = NetworkConfig{
 	GenesisEpoch:         1,
 	RegistrySyncOffset:   new(big.Int).SetInt64(84599),
 	RegistryContractAddr: "0x0d33801785340072C452b994496B19f196b7eE15",
-	AlanForkEpoch:        999999999,
-	DiscoveryProtocolID:  [6]byte{'s', 's', 'v', 'd', 'v', '5'},
+	//AlanForkEpoch:        999999999,
```

Comment:
should not be in this PR even though stage is going to be post-fork from now..

Reply:
> should not be in this PR even though stage is going to be post-fork from now..

yes, should be reverted once argo sensor is deleted

Reply:
next time just please create a separate `deploy` branch to keep the main PR clean.

Reply:
reverted